src/ucp/api/ucp.h

```diff
@@ -660,16 +732,41 @@ typedef struct ucp_ep_params {
     uint64_t                field_mask;
 
     /**
-     * Destination address; the address must be obtained using
-     * @ref ucp_worker_get_address.
-     * This field is mandatory for filling (along with its corresponding bit
-     * in the field_mask - UCP_EP_PARAM_FIELD_REMOTE_ADDRESS).
-     * The ucp_ep_create routine will return with an error if the address isn't
+     * Destination address; if a client-server connection establishment
+     * mechanism isn't used, this address must be obtained using
+     * @ref ucp_worker_get_address and this field is mandatory for filling
+     * (along with its corresponding bit in the field_mask -
+     * UCP_EP_PARAM_FIELD_REMOTE_ADDRESS).
+     * The ucp_ep_create routine will return with an error if the
+     * UCP_EP_PARAM_FIELD_SOCK_ADDR flag isn't set and this address isn't
      * specified.
      */
     const ucp_address_t     *address;
 
     /**
+     * Destination address in the form of a sockaddr; if a client-server connection
+     * establishment mechanism is used, this address must be filled and obtained
+     * from the user.
+     * The following flags need to be set:
+     * The UCP_EP_PARAM_FIELD_REMOTE_ADDRESS bit in the field_mask
+     * and the UCP_EP_PARAM_FIELD_SOCK_ADDR bit in the field_mask indicating
+     * that the type of the remote address is a sockaddr.
+     * The ucp_ep_create routine will return with an error if the
```

Comment:
this sentence should be written only in the documentation of UCP_EP_PARAM_FIELD_SOCK_ADDR flag, not here